### PR TITLE
[AppConfiguration] Add tags support when creating resource

### PR DESCRIPTION
**Pull request by shenmuxiaosen**
**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Review by avanigupta — APPROVED** [review]

**Comment by yonzhan** [comment]
AppConfiguration

File: src/azure-cli/azure/cli/command_modules/appconfig/_params.py [hunk 1]
```diff
@@ -81,6 +81,7 @@ def load_arguments(self, _):
 
     with self.argument_context('appconfig create') as c:
         c.argument('location', options_list=['--location', '-l'], arg_type=get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
+        c.argument('tags', arg_type=tags_type, help="Space-separated tags: key[=value] [key[=value] ...].")
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/appconfig/_params.py:84]: The help has been defined in `tags_type`, why do you want to define it again?

Comment (shenmuxiaosen) [reply src/azure-cli/azure/cli/command_modules/appconfig/_params.py:84]: The default help message in ```tags_type``` doesn't apply to create command. It says ```Use "" to clear existing tags```, which is suitable for update only.

Comment (zhoxing-ms) [reply src/azure-cli/azure/cli/command_modules/appconfig/_params.py:84]: OK~

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**